### ISSUE TITLE
Jupyter Notebook Alert Boxes do not appear

### ISSUE THREAD

**Issue by kevroy314** [microsoft/vscode#169945]
This is a rather odd behavior I noticed in Jupyter Notebooks being displayed in vscode. Apologies if there's a more appropriate place to put that sort of bug - I'm not sure who owns that particular module. 

I first noticed the behavior when using a Dash DataTable in Editable mode. It allows you to edit column names and uses a javascript style browser alert box to let you enter the new name. This appears normally when you run your dash application in a standard browser or even in JupyterDash in a standard Jupyter notebook, but it does not work in VSCode. I found you can reproduce the behavior much more simply like this:

In Jupyter proper (not VSCode), run:
```
from IPython.display import display, HTML
js = "<script>alert('Hello World!');</script>"
display(HTML(js))
```

You should see:
![image](https://user-images.githubusercontent.com/301111/209398561-fd9e20f5-92bc-4140-adfd-bf4929b9ff4a.png)

In VSCode Jupyter, run the same, and you won't see anything.
![image](https://user-images.githubusercontent.com/301111/209398600-11b0d017-6db1-4176-ad70-bf9af7a5e55b.png)

I imagine this has something to do with how VSCode renders the underlying page (blocking this sort of popup seems pretty reasonable). Perhaps there's a setting to disable this behavior, but I haven't been able to find it. I also couldn't really find anyone else reporting an issue like this.

**Comment by DonJayamanne** [comment]
@mjbvz I believe this is deliberate, but could you explain why exactly? 

Also is it reasonable for Jupyter extension to provide a custom implementation of `alert` so that we could re-direct the message to the extension and get the extension to display a message (this would be done in Jupyter renderer extension, if there are enough votes or we see enough users doing this).

**Comment by mjbvz** [comment]
@DonJayamanne yes  webviews in VS Code block alerts because we want extensions to use the the VS Code apis for this instead (we have no control over the native alert box)
